src/core/public/http/http_service.test.ts

```diff
@@ -6,15 +6,15 @@
  * Side Public License, v 1.
  */
 
+import { Observable } from 'rxjs';
 // @ts-expect-error
 import fetchMock from 'fetch-mock/es5/client';
-import { loadingServiceMock } from './http_service.test.mocks';
 
+import { executionContextServiceMock } from '@kbn/core-execution-context-browser-mocks';
 import { fatalErrorsServiceMock } from '@kbn/core-fatal-errors-browser-mocks';
 import { injectedMetadataServiceMock } from '@kbn/core-injected-metadata-browser-mocks';
+import { loadingServiceMock } from './http_service.test.mocks';
```

Comment:
The `{test_file}.mocks` import should always occur first-ish as is mocks packages using `doMock`, so importing other things first may mess with the jest mocking.